lib/fog/vsphere/models/compute/server.rb

```diff
@@ -237,6 +239,14 @@ def revert_snapshot(snapshot)
           end
         end
 
+        def cdroms(opts = {})
+          service.cdroms(:instance_uuid => self.id).all(opts)
+        end
```

Comment:
I guess this method is overwriting the getter defined by the `has_many_identities` DSL.

Then it does not make much sense to have the DSL and this method i guess. What do you think?